packages/python-sdk/e2b/sandbox_sync/main.py

```diff
@@ -93,12 +94,15 @@ def __init__(
         timeout: Optional[int] = None,
         metadata: Optional[Dict[str, str]] = None,
         envs: Optional[Dict[str, str]] = None,
+        secure: Optional[bool] = None,
         api_key: Optional[str] = None,
         domain: Optional[str] = None,
         debug: Optional[bool] = None,
         sandbox_id: Optional[str] = None,
         request_timeout: Optional[float] = None,
         proxy: Optional[ProxyTypes] = None,
+        envd_access_token: Optional[str] = None,
+        envd_version: Optional[str] = None,
```

Comment:
i think they shouldn't be part of `__init__`, you will never want to pass them

Reply:
I'am passing them on line 272. Originally i just extended existing call with new params. Now sure if this is proper pattern for Python.

Reply:
Moved sbx info request to init. Now params are not used.